sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -22,6 +21,7 @@ import {
   retry,
   translate,
 } from "@azure/core-amqp";
+import * as coreAmqpExperimental from "@azure/core-amqp/experimental";
```

Comment:
Wouldn't you want this not to leak into the main event hubs since the experiments can go away?  Wouldn't you want it experimental all the way down?

Reply:
Good point! You meant also have event-hubs `experimental` exports too in its beta versions?  Originally I thought that we are still doing beta version of event-hubs, and we would need to wait for core experimental feature to become GA first

Reply:
Other than another modified copy of the event hub code, it may be too involved to separate out the beta feature into experimental. /cc @deyaaeldeen